### DISCUSSION TITLE
Downstream interface not found Android 11

### DISCUSSION THREAD

**Discussion by Emtecsus** [Mygod/VPNHotspot#284] [Q&A]
![photo_2021-07-18_17-36-33](https://user-images.githubusercontent.com/69198008/126073272-7c6a0369-ab2f-4798-a1db-48ba89a80fe6.jpg)
[vpnhotspot-4656211649479810003.log](https://github.com/Mygod/VPNHotspot/files/6836525/vpnhotspot-4656211649479810003.log)

Hi I'm getting this error on the app, do you have any clue? Is it a rom problem?

**Comment by Mygod** [comment]
Thanks for the debug information. Does any other mode work? If you go to settings and click upstream interfaces, does a drop down showing valid options pop up?

**Comment by Emtecsus** [comment]
It shows wlan0

**Comment by Mygod** [comment]
Is this the case even after turning on repeater?

**Comment by Emtecsus** [comment]
Yep

**Comment by Mygod** [comment]
Thanks. Could you export debug information again when the service is active?

**Comment by Emtecsus** [comment]
[vpnhotspot-4473579118828169908.log](https://github.com/Mygod/VPNHotspot/files/6892305/vpnhotspot-4473579118828169908.log)
Sorry for keep you waiting, i couldn't do it from tablet.

**Comment by Mygod** [comment]
Thanks! Could you find the Wi-Fi after the switch is switched on?

**Comment by Emtecsus** [comment]
Yes but i'm not able to connect to it. It keeps saying "Obtaining ip address", but at the end it doesn't connect

**Comment by Mygod** [comment]
Yes this looks like a ROM problem. You should be able to connect even with the error.

**Comment by Emtecsus** [comment]
I tried again, but i can't connect to it. I'll try to report this to the dev of the rom. Thanks for the help :)

**Comment by loyukfai** [comment]
Seeing the same error, anything I can provide to troubleshoot? LOS 18.1
